gen/cloud-config.yaml

```diff
@@ -15,8 +15,6 @@ root:
     content: |
       [Journal]
       MaxLevelConsole=warning
-      RateLimitInterval=1s
-      RateLimitBurst=20000
```

Comment:
Do we really want to rely on the default or "set our default explicitly"? :) What is the default? Maybe add a code comment here?

Reply:
I'm wary of adding comments for configuration that we don't override. Like, why do we not have a comment explaining why we're using the journald default for `SyncIntervalSec`?

On the other hand, I'd like to have an explicit comment on why we set `MaxLevelConsole=warning`.